js/src/common/components/Select.tsx

```diff
@@ -0,0 +1,64 @@
+import Component, { ComponentProps } from "../Component";
+import icon from "../helpers/icon";
+
+export interface SelectProps extends ComponentProps {
+    /**u
+     * Whether or not the select is disabled.
+     */
+    disabled: boolean;
+
+    /**
+     * A callback to run when the selected value is changed.
+     */
+    onchange?: Function;
+
+    /**
+     * An object of options
+     */
+    options: {
+        [index: string]: string;
+    };
+
+    /**
+     * The current value of the select.
+     */
+    value: boolean;
+}
+
+/**
+ * The `Select` component displays a <select> input, surrounded with some extra
+ * elements for styling. It accepts the following props:
+ *
+ * - `options` A map of option values to labels.
+ * - `onchange` A callback to run when the selected value is changed.
+ * - `value` The value of the selected option.
+ * - `disabled` Disabled state for the input.
```

Comment:
Let's move these descriptions to the prop descriptions above in `SelectProps`. You already have comment documentation there, just replace them with this 🙁

Reply:
Done, good point